user_templates/printer.cfg

```diff
@@ -154,8 +154,18 @@
 # ------------------------------------------------------------------- LIGHTS AND LEDS ----
 ### --------------------------------------------------------------------------------------
 # [include config/hardware/lights/fcob_white.cfg]
+# uncomment the following line to enable neopixel caselights
 # [include config/hardware/lights/neopixel_caselight.cfg]
+# uncomment the following line to enable neopixel caselights with led effects
+# [include config/hardware/lights/neopixel_caselight_effects.cfg]
+# uncomment the following line to enable stealthburner standard 3 rgb leds
 # [include config/hardware/lights/status_leds.cfg]
+# uncomment the following line to enable stealthburner standard 3 rgb leds with led effects
+# [include config/hardware/lights/status_leds_effects.cfg]
+# uncomment the following line to enable stealthburner rainbow barf 10 rgb leds
+# [include config/hardware/lights/status_leds_rb.cfg]
+# uncomment the following line to enable stealthburner rainbow barf 10 rgb leds with led effects
+# [include config/hardware/lights/status_leds_rb_effects.cfg]
```

Comment:
@Benoitone @Frix-x i am unsure about this includes. maybe group them for led effects and one without led effects? or even have a single include for led effects? i am not sure if users would want to have one part led effect and the other not.

Reply:
I agree with you that would b ideal to have a less cluttered session, but I haven't found another way. Here we are jostling 3 things, actually 4, that may need to have effects attached: Caselights, minidisplay, and stealthburner 3leds  and rainbow barf variant. Unfortunately the effect definition doesn't accept variables for the leds settings i.e.:

[led_effect sb_logo_busy]
autostart:              false
frame_rate:             24
leds:
    neopixel:sb_leds (1-8).  ### I wish I could use a variable or alias here ###
layers:
    breathing  3 1 top (1,0,0)

This forced me to use all those includes to declare the effects literally and I left out effects for mini display not to clutter printer.cfg even further. In the end the user has the choice to use effects wherever he wants. There may be a use case where people have a ton (>100) leds for case lights and may want to use effects only on stealth burner. Never happened to me (I use 84 case lights leds) but somebody reported QGL hesitating /stopping with many leds and complex effects. If anybody has a suggestion on how to rework this to make it more palatable feel free...

Reply:
I agree that grouping into two groups of includes would be better. Something like this for example:

```
# [include config/hardware/lights/fcob_white.cfg]
# [include config/hardware/lights/neopixel_caselight.cfg]
# [include config/hardware/lights/status_leds.cfg] # Standard StealthBurner style LEDs
# [include config/hardware/lights/status_leds_rb.cfg] # Rainbow Barf StealthBurner style LEDs

# The following files are to be used with the LED effect plugin: https://github.com/julianschill/klipper-led_effect
# [include config/hardware/lights/neopixel_caselight_effects.cfg]
# [include config/hardware/lights/status_leds_effects.cfg] # Standard StealthBurner style LEDs
# [include config/hardware/lights/status_leds_rb_effects.cfg] # Rainbow Barf StealthBurner style LEDs
```